server/game/cards/08.5-TFM/Mhysa.js

```diff
@@ -0,0 +1,21 @@
+const DrawCard = require('../../drawcard.js');
+
+class Mhysa extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ trait: 'Lady' });
```

Comment:
I think it needs to be a unique Lady character.

Reply:
> I think it needs to be a unique Lady character.

Nope. "Personnage Lady uniquement" means "Lady character only".

Reply:
Aah ok. 👍 Tricksy French language!